src/app/shared/address/tf-address.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Config } from './../../config';
+import { AddressModel } from './tf-address.model';
+import { ContentfulService } from './../../services/contentful/tf-contentful.service';
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'tf-address',
+  templateUrl: 'tf-address.component.html'
+})
+
+export class AddressComponent implements OnInit {
+  addressData: AddressModel;
+
+  constructor(private contentfulService: ContentfulService) { }
+
+  ngOnInit() {
+    this.onGetAddress();
+  }
+
+  onGetAddress() {
+    let entryId = Config.Address_ENTRY;
+    this.contentfulService.getAddress(entryId)
+      .subscribe(
+      addressData => this.addressData = addressData,
```

Comment:
All Observables must be unsubscribed. 
Use the answer from this stackoverflow to better understand this. (ngunsubscribe)[https://stackoverflow.com/questions/38008334/angular-rxjs-when-should-i-unsubscribe-from-subscription]
Checkout latest talfinder project and search for keywork ngunsubscribe as a reference